content-models to review.md (#55)
SenseNet/docs.sensenet.com: content/concepts/content-management/02-content-models.md

> sensenet organizes content into content repositories, that allows you to group all the related content for a project together. Since everything is a content is sensenet this repository includes documents, workspaces, users, settings, workspaces and many more.

Reviewer: ```suggestion
sensenet organizes content into content repositories, that allows you to group all the related content for a project together. Since everything is a content in sensenet, this repository includes documents, workspaces, users, settings, workspaces and many more.
```

> Each repository has a content model, a schema that represents the content types you create. This is project-specific, so it is your own. It is important defining the business logic and creating the content types at the beginning of the project.

Author: ```suggestion
Each repository has a content model, a schema that represents the content types you create. This is project-specific, so it is your own. It is important defining the business logic and creating the content types at the beginning of the project.
    
Content model is basically the mix of schemas, content types, templates and structure of your project. Below you can learn about the properties that will form the content model of your project.
```

> # Content type hierarchy

Reviewer: nagyon hirtelen belevágunk a közepébe itt, jó lenne (ha ez megoldható) valamiféle intro ehhez a részhez. Nem érthetőek az alapok nekem itt.

Content types can inherit fields from their ancestors. For example a Domain type inherits all the fields of the basic Folder type. A content type may only inherit fields from a single type thus the content types are arranged in a simple tree hierarchy. Inherited field configuration can be overridden in derived types. Field inheritance and overriding is defined in the content type definition (CTD) of the type.

# Content naming

> In the sensenet content repository every content has a Name field that together with its location identifies the content. The content name is part of the content path, and since the path can be used to address the content via url, there are certain restrictions against the name. The content in the repository also have a DisplayName that is the user-friendly human readable name of content and can contain any kind of characters without restrictions.

Author: ```suggestion
In the sensenet content repository every content has a _Name_ field and a _DisplayName_. 
The _Name_ field together with its location is the main identifier of the content. The content name is part of the content path, and since the path can be used to address the content via url, there are certain restrictions against the name. Thus changing content’s name (aka. renaming a content) also changes the path and therefore renaming operations should be carried out carefully as it may result broken links. The content name my not contain special caracters.

The _DisplayName_ is the user-friendly human readable name of content and can contain any kind of characters without restrictions. Generally, when a content is displayed on the front-end, the value of the DisplayName property is shown. Changing the DisplayName is a simple operation and does not cause broken links (because changing only the DisplayName does not change the url).
```

> The Name is the main identifier of the content. Its value is also included in the path property which acts as a permalink to the content. Thus changing content’s name (aka. renaming a content) also changes the path and therefore renaming operations should be carried out carefully. A path change may result in a lengthy operation (paths of child content are also changed respectively) and may also result in broken links in the content repository (if another content refers to the changed one through its path - e.g. an article containing a link in its text). These two properties are used when the content is referred to via a url link and therefore may not contain special characters.

Author: ```suggestion
```

> 

Author: ```suggestion
```

> The DisplayName is the main display name of the Content. It acts as a legible, human readable name and may contain punctuations and accented characters as well. Generally, when a content is displayed on the front-end of the portal, the value of the DisplayName property is shown. Changing the DisplayName is a simple operation and does not cause broken links (because changing only the DisplayName does not change the url name).

Reviewer: ```suggestion
The DisplayName is the main display name of the Content. It acts as a legible, human readable name and may contain punctuations and accented characters as well. Generally, when a content is displayed on the front-end, the value of the DisplayName property is shown. Changing the DisplayName is a simple operation and does not cause broken links (because changing only the DisplayName does not change the url).
```

> The DisplayName is the main display name of the Content. It acts as a legible, human readable name and may contain punctuations and accented characters as well. Generally, when a content is displayed on the front-end of the portal, the value of the DisplayName property is shown. Changing the DisplayName is a simple operation and does not cause broken links (because changing only the DisplayName does not change the url name).

Author: ```suggestion
```

> Since it can be very time consuming to provide a Name and a DisplayName for a content at the same time - especially in cases when the Name can be derived from the DisplayName - sensenet provides automatic name generation mechanisms both on client and on server side. 

Reviewer: ```suggestion
Since it can be very time consuming to provide a Name and a DisplayName for a content at the same time - especially in cases when the Name can be derived from the DisplayName - sensenet provides automatic name generation mechanisms both on client and server side. 
```

# Content type properties

> The content repository contains many different types of content. Content vary in structure and even in function. Different types of content contain different fields. In sensenet built-in content types are available, but also possible to create custom ones.

Reviewer: ```suggestion
The content repository contains many different types of content. Content vary in structure and even in function. Different types of content contain different fields. In sensenet built-in content types are available, but it's also possible to create custom ones.
```

A Content Type Definition is an xml-format configuration file that holds information about:
•	the type name, description
•	properties that control how content of this type look and behave (icon, preview generation, indexing)
•	parent content type
•	set of fields (name, displayname and configuration of fields)
•	content handler

The Content Type Definition xml of a content type can be edited.

> 

Reviewer: ```suggestion

To learn more about content types, check out this [link](/concepts/content-management/03-content-types)
```

# Fields

> generic content main fields

Reviewer: ez a sor véeltelnül maradt bent? nekem nem egyértelmű

> generic content main fields

Author: ```suggestion
```
